src/api/lib/backend/test.rb

```diff
@@ -0,0 +1,47 @@
+# API for accessing to the backend
+module Backend
+  class Test
+    @backend = nil
+
+    # Starts the test backend
+    def self.start(options = {})
```

Comment:
A complete hash, you've learned from the master I guess. Why isn't it called opts?! ;-)